src/Phinx/Db/Adapter/SqlServerAdapter.php

```diff
@@ -77,6 +77,11 @@ public function connect()
             }
             $dsn .= ';database=' . $options['name'] . ';MultipleActiveResultSets=false';
 
+            // option to turn on TrustServerCertificate
+            if (isset($options['trust_server_certificate'])) {
+                $dsn .= ';TrustServerCertificate=' . $options['trust_server_certificate'];
```

Comment:
Should we handle converting the boolean into a string? Having users define a string true is a bit counter-intuitive.

Reply:
I wonder if there's a way to generalize this so that we don't need to add handlers for [all possible additional allowed options](https://docs.microsoft.com/en-us/sql/connect/php/connection-options?view=sql-server-ver15) for SQLServer. We use the prefix `sqlsrv_attr_` for detecting PDO settings, perhaps we can do a similar prefix for DSN specific settings, or have a key that expects to point at an associative array of settings? This way, we don't need to have this same if statement setup for a number of different options, and in this case, I think having them be whatever the user inputs makes sense (e.g. a string) as that's what the settings page calls for.

Reply:
Added dsn_options